test/utils.js

```diff
@@ -58,30 +58,44 @@ module.exports.promiseSetupDriver = async() => {
     .forBrowser("firefox")
     .setFirefoxOptions(options);
 
-  const binaryLocation = await promiseActualBinary(process.env.FIREFOX_BINARY || "firefox");
+  const binaryLocation = await promiseActualBinary(
+    process.env.FIREFOX_BINARY || "firefox",
```

Comment:
I would not expect a comma here after the first arg. Seems like Prettier is getting confused?

Reply:
Maybe, but leaving as is since changing this line here requires major rebasing and conflict handling in the commits of #120 so I suggest that this is merged as is and if it is still a problem after #120, it should be cleaned up in the develop branch later.

Reply:
Does this sound related? https://github.com/prettier/prettier/issues/2788, and if so, should we change https://github.com/mozilla/shield-studies-addon-utils/pull/119/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R52 and we should maybe change `--trailing-comma=all` to `--trailing-comma=es5`?

> #### Trailing Commas
> Print trailing commas wherever possible when multi-line. (A single-line array,
for example, never gets trailing commas.)
>
> Valid options:
>
> - "none" - No trailing commas.
> - "es5" - Trailing commas where valid in ES5 (objects, arrays, etc.)
> - "all" - Trailing commas wherever possible (including function arguments). This requires node 8 or a transform.